tests/test_observatory.py

```diff
@@ -1,18 +1,46 @@
 #!/usr/bin/env python
+import io
 import os
+from pathlib import Path
 import unittest
 
 import numpy as np
 import pytest
 from pint.pulsar_mjd import Time
 
 import pint.observatory
-import pint.observatory.observatories
+import pint.observatory.topo_obs
+from pint.observatory.topo_obs import (
+    load_observatories,
+    load_observatories_from_usual_locations,
+    TopoObs,
+)
+from pint.observatory.special_locations import load_special_locations
 from pint.observatory import get_observatory, Observatory, NoClockCorrections
-from pint.observatory.topo_obs import TopoObs
 from pinttestdata import datadir
 
 
+@pytest.fixture
+def sandbox():
+    class Sandbox:
+        pass
+
+    o = Sandbox()
+    e = os.environ.copy()
+
+    try:
+        del os.environ["PINT_OBS_OVERRIDE"]
+    except KeyError:
+        pass
+    reg = pint.observatory.Observatory._registry.copy()
```

Comment:
If you've done this, you can just assign it back and avoid the reloading - potentially a help is a bug arises in the loading. Or you can ditch this line and rely on reloading to get the registry straightened out.